### PR TITLE
[12.0][IMP] account_financial_risk: Add allow overrisk invoice validation option.

### PR DESCRIPTION
**Pull request by carlosdauden**
Improve sales cases, views and translations.

@Tecnativa

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Tests aren't happy

File: account_financial_risk/models/res_company.py [hunk 1]
```diff
@@ -13,3 +13,9 @@ class ResCompany(models.Model):
              "The change of this field recompute all partners risk, "
              "be patient.",
     )
+    allow_overrisk_invoice_validation = fields.Boolean(
+        string="Allow overrisk invoice validation",
```
Comment (pedrobaeza) [inline account_financial_risk/models/res_company.py]:
```suggestion
        string="Allow invoice validation over the risk",
```

File: account_financial_risk/views/res_config_view.xml [hunk 2]
```diff
@@ -8,19 +8,41 @@
...
-        <xpath expr="//div[@id='account_followup']/.." position="inside">
-            <div class="col-12 col-lg-6 o_setting_box" title="Days after due date to set an invoice as unpaid. The change of this field recompute all partners risk, be patient.">
-                <div class="o_setting_left_pane"/>
-                <div class="o_setting_right_pane">
-                    <span class="o_form_label">Financial Risk</span>
-                    <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." aria-label="Values set here are company-specific." groups="base.group_multi_company" role="img"/>
-                    <div class="text-muted">
-                        Set options applied to financial risk
+        <xpath expr="//div[@id='account_followup']/.." position="after">
+<!--                        <h2>Customer Payments</h2>-->
```
Comment (pedrobaeza) [inline account_financial_risk/views/res_config_view.xml]: ?

**Comment by carlosdauden** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
:+1:

**Pull request merged**